card/lib/card/format/nest.rb

```diff
@@ -12,6 +12,11 @@ def subformat subcard
                        mainline: @mainline, form: @form
       end
 
+      def field_subformat field
+        field = card.cardname.field(field) unless field.is_a?(Card)
+        subformat field
+      end
```

Comment:
Are there better existing options to render nests for fields in a view?
With this I can do something like `field_subformat('about')._render_content` to get the same result that `{{+about}}` would produce